docs/data/base/getting-started/accessibility/accessibility.md

```diff
@@ -0,0 +1,58 @@
+# Accessibility
+
+<p class="description">
+Learn how to make the most of Base UI's accessibility features and guidelines.
+</p>
+
+Accessibility is our top priority for Base UI components.
+However, the library cannot make your application fully accessible on its own.
+This page highlights the most common actions that you as a developer must take to ensure that your app is accessible to users who require assistive technologies.
+
+## Keyboard navigation
+
+Base UI components follow the [WAI-ARIA 1.2 standard](https://www.w3.org/TR/wai-aria-1.2/), so they are accessible with a keyboard out of the box.
+This is important for users who have trouble using a pointing device, but also comes in handy for users who find navigating with a keyboard faster, and who expect web components to behave the same way as native operating system controls.
+
+Specifically, interactive components can be focused using the <kbd>Tab</kbd> key.
+List-like components (such as Select and Menu) can be browsed and activated using arrow keys, <kbd>Home</kbd>, <kbd>End</kbd>, <kbd>Enter</kbd>/<kbd>Return</kbd>, and <kbd>Escape</kbd>.
```

Comment:
```suggestion
List-like components (such as Select and Menu) can be browsed and activated using arrow keys, <kbd class="key">Home</kbd>, <kbd class="key">End</kbd>, <kbd class="key">Enter</kbd>/<kbd class="key">Return</kbd>, and <kbd class="key">Escape</kbd>.
```

Adding the `class="key"` declaration as I noticed the styles were off on the deploy preview. 😬 
Sorry for not including this in the previous review!